site/content/en/guides/producer/functions/exec/_index.md

```diff
@@ -62,50 +68,58 @@ metadata:
         path: /path/to/executable
 ```
 
-Note: if the `--enable-exec` flag is not provided, `kpt fn run DIR/` will ignore the exec
-function and exit 0.
+Note: if the `--enable-exec` flag is not provided, `kpt fn run DIR/` will
+ignore the exec function and exit 0.
 
 ## Typescript Function Example
 
-You may want to run a function developed with one of the config function SDKs using the exec
-runtime in order to avoid the overhead associated with running a container. To run your function
-in the exec runtime, you will first need to package your function as an executable.
+You may want to run a function developed with one of the config function SDKs
+using the exec runtime in order to avoid the overhead associated with running
+a container. To run your function in the exec runtime, you will first need to
+package your function as an executable.
 
-We walk through an example of running a typescript function using the kpt exec runtime.
+We walk through an example of running a typescript function using the kpt exec
+runtime.
 
 ### Prerequisites
 
-* Install the pkg CLI.
+- Install the pkg CLI.
 
-    ```sh
-    npm install -g pkg
-    ```
+  ```sh
+  npm install -g pkg
+  ```
 
-* Install your kpt-functions package module to create your function's distributable file.
+- Install your kpt-functions package module to create your function's
+  distributable file.
 
-    ```sh
-    npm i
-    ```
+  ```sh
+  npm i
+  ```
 
 ### Steps
 
-1. Use the pkg CLI to create an executable from your function's distributable file. For a my_func
-   function built using the typescript SDK, this is `dist/my_func_run.js`.
+1. Use the pkg CLI to create an executable from your function's distributable
+   file. For a `my_fn` function built using the typescript SDK, this is
+   `dist/my_fn_run.js`.
 
-    ```sh
-    npx pkg dist/my_func_run.js
-    ```
+   ```sh
+   npx pkg dist/my_fn_run.js
+   ```
 
-2. Pass the path to the appropriate executable for your OS when running kpt using the exec runtime.
+1. Pass the path to the appropriate executable for your OS when running kpt
+   using the exec runtime.
 
-    ```sh
-    kpt fn run DIR/ --enable-exec --exec-path /path/to/my_func_run-macos -- foo=bar baz=qux
-    ```
+   ```sh
+   kpt fn run DIR/ --enable-exec --exec-path /path/to/my_fn_run-macos -- a=b
+   ```
 
 ## Next Steps
 
-* Find out how to structure a pipeline of functions from the [functions concepts] page.
-* Consult the [fn command reference].
+- Explore other ways to run functions, such as using the [starlark runtime].
```

Comment:
Not sure this is the right next step. I think there should be a top-level docs page discussing different runtimes and languages, and this should link to that.

Reply:
Updated to link to the Functions Developer Guide, which discusses different runtimes and languages. Also updated the other runtime pages to link to the Functions Developer Guide.